tfe/data_source_ip_ranges_test.go

```diff
@@ -0,0 +1,38 @@
+package tfe
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccTFEIPRangesDataSource_basic(t *testing.T) {
+	ipRegex := regexp.MustCompile(`^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\/([1-9]|[1-9][0-9])$`)
```

Comment:
Nit: I love a good complicated regex, but since we really only care that this _is_ an IP address and don't necessarily care about ensuring that it's valid (that's the API's job!), what if we made this a little fuzzier and had it just match the format of an IPv4 address?

```suggestion
	ipRegex := regexp.MustCompile(`^[\d]{1,3}\.[\d]{1,3}\.[\d]{1,3}\.[\d]{1,3}/[\d]{1,3}$`)
```

Reply:
Totally! TBH I didn't even write this regex, it's just cribbed because I was lazy 😜  Yours is very reasonable and what I should have done to start 👍🏻

Solid feedback.